### ISSUE TITLE
Pyinstaller packaging not working for version 4.52

### ISSUE THREAD

**Issue by jowlo** [fonttools/fonttools#3531]
When updating an application that is packaged using pyinstaller to run on windows, upon application start the new logic to support the `VARC` tables produces this load time error.

```
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "fontTools\subset\__init__.py", line 2634, in <module>
  File "fontTools\subset\util.py", line 14, in wrapper
AssertionError: Oops, table class not found.
```

While i haven't investigated too far down, this is most likely related due to pyinstaller not being able to automatically find dynamically imported classes and not packaging them correctly at "build" time.

This error can be fixed here or in the pyinstaller project (as a new hooks in the [pyinstaller-hooks-contrib repository](https://github.com/pyinstaller/pyinstaller-hooks-contrib)). However, I struggle to find which classes are dynamically loaded in this case.

Downgrading to fonttools to 4.51 does not produce the issue.

**Comment by jowlo** [comment]
Oh, nice! My main motivation to open the issue was that I couldn't find anyone else running into this error, but i guess a few people will in the next days.

I guess most projects have this package as some transitive dependency (from _fpdf2_ in my case) and being able to google that error message hopefully helps.
 
Thanks for the incredibly quick fix and response!

**Comment by justvanrossum** [comment]
Right, it is clear that I underestimated the reach of this problem. @anthrotype, can you trigger a release?

**Comment by jowlo** [comment]
FWIW, when using 369a213 my project is packaged correctly and runs just as expected.

**Comment by anthrotype** [comment]
I’m away from laptop right now, I can do it tomorrow, or else @khaledhosny can do

**Comment by khaledhosny** [comment]
I just released [4.52.2](https://github.com/fonttools/fonttools/releases/tag/4.52.2), should hit PyPI soon (hopefully).
